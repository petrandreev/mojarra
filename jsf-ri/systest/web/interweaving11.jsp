<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>

<%@ page contentType="text/html;charset=UTF-8" language="java" %>
<%
    java.util.ArrayList list = new java.util.ArrayList();
    list.add("one");
    list.add("two");
    list.add("three");
    pageContext.setAttribute("list", list, PageContext.REQUEST_SCOPE);
%>
<html>
  <head>
      <title>interweaving11</title>   
  </head>
  <body>
    <f:view>
        <h:form>
            <c:forEach items="#{list}" var="item">
                <h:commandLink>
                   <f:param name="param" value="value"/>
                   <h:outputText value="#{item}"/>
                </h:commandLink>
                </br>
            </c:forEach>
        </h:form>
    </f:view>
  </body>
</html>

