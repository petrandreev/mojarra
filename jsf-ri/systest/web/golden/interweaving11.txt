





<html>
  <head>
      <title>interweaving11</title>   
  </head>
  <body>
    
<form id="j_id_id17" name="j_id_id17" method="post" action="/jsf-systest/faces/interweaving11.jsp;jsessionid=ad5061b1858936d02ffb9e30cf8a" enctype="application/x-www-form-urlencoded">
<input type="hidden" name="j_id_id17" value="j_id_id17" />
<input type="hidden" name="javax.faces.ViewState" id="javax.faces.ViewState" value="j_id39:j_id40" />

<script type="text/javascript" language="Javascript">
<!--
for (var f = 0; f < document.forms.length; f++) {
    var fm = document.forms[f];
    fm.dpf = function() {
        var adp = this.adp;
        if (adp != null) {
            for (var i = 0; i < adp.length; i++) {
                this.removeChild(adp[i]);
            }
        }
        this.adp = null;
    };
    var mos = fm.onsubmit;
    if (mos == null) {
        fm.onsubmit = fm.dpf;
    } else {
        fm.onsubmit = function() {
            fm.dpf();
            mos();
        }
    }
    fm.apf = function(pvp) {
        this.dpf();
        var adp = new Array();
        this.adp = adp;
        var ps = pvp.split(',');
        for (var i = 0, ii = 0; i < ps.length; i++, ii++) {
            var p = document.createElement("input");
            p.type = "hidden";
            p.name = ps[i];
            p.value = ps[i + 1];
            this.appendChild(p);
            adp[ii] = p;
            i += 1;
        }
    };
}
function jsfcljs(f, pvp, t) {
    if (f.apf) {
        f.apf(pvp);
    }
    if (t) {
       f.target = t;
    }
    f.submit();
    return false;
};
//-->
</script>
<a href="#" onclick="return jsfcljs(document.forms['j_id_id17'],'j_id_id17:j_id_id21,j_id_id17:j_id_id21,param,value','');">one</a>
            
                
                </br>
            
                <a href="#" onclick="return jsfcljs(document.forms['j_id_id17'],'j_id_id17:j_id_id21j_id_1,j_id_id17:j_id_id21j_id_1,param,value','');">two</a>
                </br>
            
                <a href="#" onclick="return jsfcljs(document.forms['j_id_id17'],'j_id_id17:j_id_id21j_id_2,j_id_id17:j_id_id21j_id_2,param,value','');">three</a>
                </br>
            
        
</form>
  </body>
</html>

